docs/standards/universal-profile/06-lsp5-received-assets.md

```diff
@@ -1,6 +1,6 @@
 ---
 sidebar_label: 'LSP5 - Received Assets'
-sidebar_position: 6
+sidebar_position: 5
```

Comment:
says 5 but filename says 6 - i suggest to remove the `06` from the filename

Reply:
as above- will tackle structure in separate PR as it changes all the sidelinks